lib/command.ts

```diff
@@ -262,15 +270,20 @@ export function command(
       // the options that a command takes.
       const innerYargs = yargs.reset(parsed.aliases);
       if (shouldUpdateUsage(innerYargs)) {
-        innerYargs
-          .getUsageInstance()
-          .usage(
-            usageFromParentCommandsCommandHandler(
-              parentCommands,
-              commandHandler
-            ),
-            commandHandler.description
-          );
+        if (!command) {
+          tmpCommands.forEach(e => innerYargs.getUsageInstance().command(...e));
```

Comment:
Now that I dug into the problem, this was 95% of the way there. The advantage of calling `.unfreeze()` instead is it will give you _exactly_ the output of usage at the top level.